core/jvm/src/main/scala/cats/effect/unsafe/Repl.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+package unsafe
+
+object Repl {
+  implicit class Yolo[A](private val self: IO[A]) extends AnyVal {
```

Comment:
🤔 What do you think about also adding a `Yolo` for `SyncIO` ?

Reply:
Definitely (once we have `SyncIO` on this branch)

Reply:
I actually think `SyncIO` is a lot safer to yolo your way to victory, since it doesn't involve any executors, so you really can have a "safe" `unsafeRunSync()` that doesn't take any parameters. It still runs side-effects, obviously, but it's at least not going to leak resources.